src/hooks/useThrottledButtonState.ts

```diff
@@ -1,9 +1,8 @@
 import {useEffect, useState} from 'react';
 
-/**
- * @returns {Array}
- */
-export default function useThrottledButtonState() {
+type ThrottledButtonState = [boolean, () => void];
```

Comment:
```suggestion
type ThrottledButtonState = [boolean, Dispatch<SetStateAction<boolean>>];
```

Reply:
@kubabutkiewicz when I apply your suggestion, I'm getting this tooltip:

![image](https://github.com/Expensify/App/assets/85628901/7e5c5c60-8298-4fec-a35d-790fcf6d8b6a)

Which is expecting a boolean value to be passed to that function. In reality though, this function doesn't accept any params. LMK your thoughts.

Reply:
ok, I thought we are returning the state setter from this hook but we are returning function which is just setting state to `false` so nvm. LGTM